backend/papersfeed/constants.py

```diff
@@ -9,7 +9,7 @@
 NOT_EXIST_OBJECT = 404  # 존재하지 않는 Object
 USERNAME_ALREADY_EXISTS = 419  # 이미 존재하는 사용자 이름
 EMAIL_ALREADY_EXISTS = 420  # 이미 존재하는 이메일
-
+INVALID_JSON = 421  # json.loads 에러
```

Comment:
Because 4XX means client errors and 5XX means server errors(https://restfulapi.net/http-status-codes/), I think a customized 5XX(like 520) would be better.

Reply:
520으로 수정하였습니다.